cypress/page/personal-form.page.ts

```diff
@@ -0,0 +1,72 @@
+interface PersonalInformation {
+    name: string;
+    lastName: string;
+    email: string;
+    gender: string;
+    dateOfBirth: string;
+    mobileNumber: string;
+    hobbies: string[];
+    currentAddress: string;
+}
+
+class PersonalFormPage {
+    private personalInfoFormUrl: string;
+    private nameInput: string;
+    private lastnameInput: string;
+    private emailInput: string;
+    private genderRadioBtn: string;
+    private dateOfBirthPicker: string;
+    private datePickerValues: string[];
+    private mobileNumberInput: string;
+    private hobbiesCheckbox: string;
+    private currentAddressInput: string;
+    private submitBtn: string;
+    private modalTitle: string;
+
+    constructor() {
+        this.personalInfoFormUrl = "https://demoqa.com/automation-practice-form";
+        this.nameInput = "#firstName";
+        this.lastnameInput = "#lastName";
+        this.emailInput = "#userEmail";
+        this.genderRadioBtn = '[name="gender"]';
+        this.dateOfBirthPicker = "#dateOfBirthInput";
+        this.datePickerValues = [".react-datepicker__year-select", ".react-datepicker__month-select", ".react-datepicker__day"];
```

Comment:
could you convert this into a dictionary, instead of an array, so when you refer to the selectors you can use dot notation instead of indexes